ProjectLighthouse.Servers.Website/Controllers/Admin/ModerationSlotController.cs

```diff
@@ -27,6 +28,16 @@ public async Task<IActionResult> TeamPick([FromRoute] int id)
         if (slot == null) return this.NotFound();
         slot.TeamPick = true;
 
+        // Send webhook with slot.Name and slot.Creator.Username
+        if (slot.Creator != null)
+        {
+            await WebhookHelper.SendWebhook("Team Pick", $"Level {slot.Name} by {slot.Creator.Username} has been marked as a team pick.");
+        }
+        else
+        {
+            await WebhookHelper.SendWebhook("Team Pick", $"Level {slot.Name} has been marked as a team pick.");
+        }
+
```

Comment:
```suggestion
        await WebhookHelper.SendWebhook("New Team Pick!", $"The level [**{slot.Name}**]({ServerConfiguration.Instance.ExternalUrl}/slot/{slot.SlotId}) by **{slot.Creator?.Username}** has been team picked");
```
Since we used Include() in the slot fetch above there is never a case where the Creator can be null so we don't need the if statement anymore.
Another small nitpick is the message could have a link to the slot that got team picked and I also added emphasis to the creator's name by bolding it